src/Integrations/Mailer/SettingsMailer.php

```diff
@@ -214,157 +226,181 @@ public function getSettingsData(string $formId): array
-								// translators: %s will be replaced with forms field name.
-								'textareaFieldHelp' => \sprintf(\__('
-									Use template tags to use submitted form data (e.g. <code>{field-name}</code>)
-									<details class="is-filter-applied">
-										<summary>Available tags</summary>
-										<ul>
-											%1$s
-										</ul>
-
-										<br />
-										Tag missing? Make sure its field has a <b>Name</b> set!
-									</details>', 'eightshift-forms'), $formNames),
-								'textareaIsRequired' => true,
-								'textareaValue' => $this->getSettingsValue(self::SETTINGS_MAILER_TEMPLATE_KEY, $formId),
-							],
-						]
-					],
-					[
-						'component' => 'tab',
-						'tabLabel' => \__('Confirmation e-mail', 'eightshift-forms'),
-						'tabContent' => [
-							[
-								'component' => 'intro',
-								'introSubtitle' => \__('Sent to the user that filled in the form, e.g. a "thank you" message.', 'eightshift-forms'),
-							],
-							[
-								'component' => 'divider',
-								'dividerExtraVSpacing' => 'true',
-							],
-							[
-								'component' => 'checkboxes',
-								'checkboxesFieldLabel' => '',
-								'checkboxesName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_USE_KEY),
-								'checkboxesContent' => [
-									[
-										'component' => 'checkbox',
-										'checkboxLabel' => \__('Use confirmation email', 'eightshift-forms'),
-										'checkboxIsChecked' => $isSenderUsed,
-										'checkboxValue' => self::SETTINGS_MAILER_SENDER_USE_KEY,
-										'checkboxSingleSubmit' => true,
-										'checkboxAsToggle' => true,
-									]
-								]
+					...($emailField ? [
+						[
+							'component' => 'tab',
+							'tabLabel' => \__('Advanced', 'eightshift-forms'),
+							'tabContent' => [
+								[
+									'component' => 'input',
+									'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_NAME_KEY),
+									'inputFieldLabel' => \__('Email client sender name', 'eightshift-forms'),
+									'inputFieldHelp' => \__('Most e-mail clients will show this instead of the e-mail address in the list of e-mails. Make this something related to your brand that will distinguish you from the rest.', 'eightshift-forms'),
+									'inputType' => 'text',
+									'inputIsRequired' => true,
+									'inputValue' => $this->getSettingsValue(self::SETTINGS_MAILER_SENDER_NAME_KEY, $formId),
+								],
+								[
+									'component' => 'divider',
+									'dividerExtraVSpacing' => 'true',
+								],
+								[
+									'component' => 'input',
+									'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_EMAIL_KEY),
+									'inputFieldLabel' => \__('Email client from email', 'eightshift-forms'),
+									'inputFieldHelp' => \__('Most e-mail clients will us this as field as the senders email and will use it when the user clicks on the respond to button. Make this something like: `info@infinum.com`.', 'eightshift-forms'),
```

Comment:
```suggestion
									'inputFieldHelp' => \__('Most e-mail clients will us this as field as the sender email (displayed as <i>From:</i>). It will also be used as a reply-to destination. Example: `info@infinum.com`.', 'eightshift-forms'),
```